### DISCUSSION TITLE
Faculty Position - Coastal Modeler - Oregon State University, Oregon, USA

### DISCUSSION THREAD

**Discussion by dfosterhill** [CliMA/Oceananigans.jl#3647] [General]
Colleagues.

Please see information below re: a great new position here at Oregon State University. Come join some great people in a great location…

*****

The School of Civil and Construction Engineering (CCE) at Oregon State University (OSU) seeks to fill one full-time (1.00 FTE), 9-month, tenure-track or tenured position, at the rank of Assistant, or Associate, or Full Professor in the area of Coastal and Ocean Engineering. The successful candidate will have active research in coastal ocean or nearshore computational modeling including hydrodynamics, fluid-structure, fluid-sediment, or fluid-ice interactions. Potential modeling applications include a range of scales from regional to local, could consider interactions with the built and natural environment, impacts of climate change, and help find solutions to coastal problems. The successful candidate will leverage High Performance Computing capabilities including GPU-based techniques at OSU. Educational priorities will include undergraduate teaching and graduate-level training and advising in the use of computational tools focused on the coastal zone; courses could be delivered in-person, hybrid, or via eCampus.  

 

The OSU coastal and ocean engineering and coastal processes community consists of a vibrant group of faculty, students, post-docs, and technical research staff. The new Jen-Hsun Huang and Lori Mills Huang Collaborative Innovation Complex will support collaborative research and will home the Nvidia Earth-2 Super Computer (https://leadership.oregonstate.edu/huang-cic) supporting high-performance GPU based computing. Additionally, Oregon State has a highly active coastal campus, the Hatfield Marine Science Center, located on the Pacific Coast in Newport, OR that interfaces with several federal agencies.  

 

Oregon State’s beautiful, historic and state-of-the-art main campus is located in one of America’s best college towns. Corvallis is located close to the Pacific Ocean, the Cascade mountains and Oregon wine country. Nestled in the heart of the Willamette Valley, this beautiful city offers miles of mountain biking and hiking trails, a river perfect for boating or kayaking and an eclectic downtown featuring local cuisine, popular events and performances.  

 

To learn more and apply, please visit: https://jobs.oregonstate.edu/postings/156459  

 

The position is open for full consideration until August 15th 2024 and will close on August 31st 2024. 

David F. Hill
Professor
Civil and Construction Engineering
Oregon State University
207 Owen Hall
Corvallis, OR 97331
541.737.4939
[david.hill@oregonstate.edu](mailto:david.hill@oregonstate.edu)
http://www.engr.oregonstate.edu/~dfh

**Comment by glwagner** [comment]
Love this use of a discussion! Maybe we should create a new category for "jobs". Folks should tag people here who might be interested.
